circuit-benchmarks/src/bench_params.rs

```diff
@@ -0,0 +1 @@
+pub(crate) const DEGREE: usize = 19;
```

Comment:
This new file seems unrelated to this PR

Reply:
I agree this change should not be included, please